cli/tools/bundle/provider.rs

```diff
@@ -0,0 +1,193 @@
+// Copyright 2018-2025 the Deno authors. MIT license.
+
+use std::sync::Arc;
+
+use deno_core::error::AnyError;
+use deno_runtime::ops::bundle as rt_bundle;
```

Comment:
Maybe we could move it to a separate `ext/` crate and then we could get rid of structs like `crate::args::BundleFormat` or `crate::args::BundlePlatform` and use a single set of enums